jobQueue/jobWrapper.js

```diff
@@ -126,11 +126,10 @@ function logJobOnComplete(req, job) {
       }
 
       logObject.ipAddress = activityLogUtil.getIPAddrFromReq(req);
-      jwtUtil.getTokenDetailsFromRequest(req)
-      .then((resObj) => {
-        logObject.user = resObj.username;
-        logObject.token = resObj.tokenname;
-      }).catch(() => {}); // no-op
+      const tokenDetails = jwtUtil.getTokenDetailsFromRequest(req);
+      logObject.user = tokenDetails.userName;
```

Comment:
getTokenDetailsFromRequest is still asynchronous, correct?
It calls getTokenDetailsFromTokenString, which returns a promise.